qiita_db/parameters.py

```diff
@@ -172,6 +172,27 @@ def to_str(self):
 
         return " ".join(result)
 
+    @classmethod
+    def get_all_parameters(self):
```

Comment:
In other places (e.g., for the [user object](https://github.com/biocore/qiita/blob/master/qiita_db/user.py#L90-L104) or the [study person object](https://github.com/biocore/qiita/blob/master/qiita_db/study.py#L799-L815)) we have an `iter` method that accomplishes a similar goal, I think; perhaps follow that model?